File: speed_test.c
#include <stdio.h>

int isPrime(int n){
    for (int i=2;i<=n/2;i++)
        if (!(n%i))
        return 0;
    return 1;
}

int main(){
    int numPrimes=0;

    for (int i=2;i<250001;i++)
        numPrimes+=isPrime(i);

    printf("%d\n",numPrimes);
    printf("finish\n");
}

File: speed_test.py
def isPrime(n):
    for i in range(2,n//2+1):
        if (not (n%i)):
            return 0;
    return 1;

numPrimes=0;

for i in range(2,250001):
    numPrimes+=isPrime(i);
    
print(str(numPrimes));
print("finish");